web/src/containers/ApdPreviousActivityTable.js

```diff
@@ -78,37 +79,51 @@ const ApdPreviousActivityTable = ({
 
               <td
                 headers={`prev_act_hithie_row_${year} prev_act_hithie_total prev_act_hithie_total_approved`}
+                className={isViewOnly ? 'budget-table--number' : ''}
               >
-                <DollarField
+                {isViewOnly ?
+                  <Dollars long>
```

Comment:
Oooh, that makes sense! Being able to have the app itself keep that context (pun intended) rather than having each component would be way cleaner. Theoretically, at least. 😆